dask/bytes/tests/test_s3.py

```diff
@@ -461,6 +461,11 @@ def test_parquet(s3, engine, s3so, metadata_file):
     pd = pytest.importorskip("pandas")
     np = pytest.importorskip("numpy")
 
+    from dask.array.numpy_compat import NUMPY_GE_200
+
+    if NUMPY_GE_200 and engine == "fastparquet":
+        pytest.skip("fastparquet doesn't work with Numpy 2")
```

Comment:
Would you mind leaving a comment pointing to the upstream issue so we know when we can remove these skips?